components/Item/Card/Card.vue

```diff
@@ -71,7 +71,14 @@
       </div>
       <div v-if="text" class="card-text">
         <div class="card-title mt-1 text-truncate">{{ cardTitle }}</div>
-        <div class="card-subtitle text--secondary text-truncate">
+        <nuxt-link
+          v-if="item.Type === 'MusicAlbum'"
+          class="card-subtitle text--secondary text-truncate link card-subtitle-link"
+          :to="getItemDetailsLink(item.AlbumArtists[0], 'MusicArtist')"
+        >
+          {{ cardSubtitle }}
+        </nuxt-link>
```

Comment:
```suggestion
        <nuxt-link
          v-if="item.Type === 'MusicAlbum'"
          tag="div"
          class="card-subtitle text--secondary text-truncate link"
          :to="getItemDetailsLink(item.AlbumArtists[0], 'MusicArtist')"
        >
          {{ cardSubtitle }}
        </nuxt-link>
```

Maybe this would fix it? We technically lose the link display on the bottom left, but I'm not really sure that matters.

Otherwise, no need to make a custom class, you can use [Vuetify's display utility classes](https://vuetifyjs.com/en/styles/display/). In this case, `d-block` should do the trick :wink: 

I'm also wondering if this is really usable on mobile :thinking: I can test it later today with my fat fingers to see if it's usable or not. If not, it's pretty easy to disable it on mobile.